controllers/patient.controller.js

```diff
@@ -21,4 +31,27 @@ const post = async (req, res) => {
   }
 };
 
-module.exports = { post, get };
+const update = async (req, res) => {
+  try {
+    const value = await patientSchema.validateAsync(req.body);
```

Comment:
Where is `id` of the patient? You should pass id in url params and read it from there. And the for the update service pass like:
```
serivce.update(id, data)
```